wikipron/languagecodes.py

```diff
@@ -257,5 +257,57 @@
     # Moroccan Arabic. ISO 639-3 only.
     "ary": "Moroccan Arabic",
     # Mandarin Chinese. ISO 639-3 only.
-    "cmn": "Chinese"
+    "cmn": "Chinese",
+    # Abkhaz. Would be Abkhazian in ISO 639.
+    "abk": "Abkhaz",
+    # Avar. Would be Avaric in ISO 639.
+    "ava": "Avar",
+    # Buryat. Would be Buriat in ISO 639.
+    "bua": "Buryat",
+    # Chukchi. Would be Chukot in ISO 639.
+    "ckt": "Chukchi",
+    # Burushaski. ISO 639-3 only.
+    "bsk": "Burushaski",
+    # Galician. Not in iso639 lib.
+    "glc": "Galician",
```

Comment:
One issue I've noticed: `glc` is for Bon Gula, not Galician. For Galician, the ISO 639-2 or -3 code is `glg`. FWIW, we've already scraped the Galician data from Wikitionary, see [here](https://github.com/kylebgorman/wikipron/tree/7ea49af5802a60208382fcac17d5855dae56376f/languages/wikipron).

Reply:
Oh, my bad. Will remove this now.